### PR TITLE
Responsive single post

### LINKED ISSUE #15: Responsive single post

### PR DESCRIPTION
**Pull request by Eliassoren**
Fixes #15

### CODE REVIEW

**Comment by okaune** [comment]
You can remove the current hover function on the post card altogether, leave the css block empty and we will make a better hover effect soon :)

File: resources/scss/components/_post-card.scss [hunk 1]
```diff
@@ -48,4 +48,11 @@
     font-size: .625rem;
     color: $light-gray;
   }
+  @media(max-width: 900px){
+    &:hover {
+      z-index: 1;
+      transform: scale(1);
+    }
+    transition: none;
```
Comment (okaune) [inline resources/scss/components/_post-card.scss]: Transition doesn't need to be reset if not used

File: resources/scss/components/_right-sidebar.scss [hunk 2]
```diff
@@ -3,15 +3,31 @@
   &__header{
     width: 100%;
     display: block;
+    @media(max-width: 900px){
+      border: .3rem solid $blue;
```
Comment (okaune) [inline resources/scss/components/_right-sidebar.scss]: Why blue border?

File: resources/scss/components/_right-sidebar.scss [hunk 3]
```diff
@@ -3,15 +3,31 @@
...
+    }
   }
   .post-card{
     width: 100%;
     background: $white;
     color: $black;
+    border: .3rem solid $blue;
```
Comment (okaune) [inline resources/scss/components/_right-sidebar.scss]: Why blue border?

File: resources/scss/components/_right-sidebar.scss (hunk 2 above)
Comment (Eliassoren) [reply resources/scss/components/_right-sidebar.scss]: Quickfix replacement of padding, which solved this problem:

noborder

blueborder

**Pull request merged**